newmiddleware/formsg.ts

```diff
@@ -0,0 +1,24 @@
+import autoBind from "auto-bind"
+import express, { Request, Response, NextFunction } from "express"
+
+import FormSGService from "@root/services/middlewareServices/FormSGService"
+
+export default class FormSGMiddleware {
+  formSGService: FormSGService
+
+  constructor({ formSGService }: { formSGService: FormSGService }) {
+    this.formSGService = formSGService
+    // We need to bind all methods because we don't invoke them from the class directly
+    autoBind(this)
+  }
```

Comment:
actually i just realised all of this is known ahead of time as the service doesn't take anything at runtime also. we might want to just delete this/shift the service here/???

Reply:
As above, removing this class!